md/Depth-Limited-Search.md

```diff
@@ -1,5 +1,28 @@
 # DEPTH-LIMITED-SEARCH
 
+## AIMA4e  
+
+__function__ DEPTH-LIMITED-SEARCH(_problem_, _l_) __returns__ a solution, or failure, or cutoff  
+&emsp;_frontier_ &larr; a FIFO queue initially containing one path, for the _problem_'s initial state  
+&emsp;_solution_ &larr; failure  
+&emsp;__while__  _frontier_ is not empty __do__  
+&emsp;&emsp;&emsp;_parent_ &larr; pop(_frontier_)  
+&emsp;&emsp;&emsp;__if__ depth(_parent_) > l __then__  
+&emsp;&emsp;&emsp;&emsp;&emsp;_solution_ &larr; cutoff  
+&emsp;&emsp;&emsp;&emsp;&emsp;__else__  
+&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;__for__ _child_ __in__ successors(_parent_) __do__  
+&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;__if__ _child_ is a goal __then__  
```

Comment:
@norvig 
In the rest of the chapter, we check if a state is a goal state or not. However, in this algorithm, we have applied the test directly on a node. The test is applied as follows in the rest of the algorithms:
&emsp;__for__ _child_ __in__ successors(_parent_) __do__  
&emsp;&emsp;&emsp; _s_ &larr; _child_.state  
&emsp;&emsp;&emsp;__if__ _s_ is a goal  __then__  
and then the rest of the code. Please have a look into this.

Reply:
I think it should be child.state here too.